lib/Parse/ParseStmt.cpp

```diff
@@ -21,6 +21,7 @@
 #include "clang/Sema/PrettyDeclStackTrace.h"
 #include "clang/Sema/Scope.h"
 #include "clang/Sema/TypoCorrection.h"
+#include "clang/Sema/SemaDiagnostic.h"
```

Comment:
Is this new include needed in this file?   I am not seeing changes to this file that seem to use the result of the include.